DIPs/DIP1012.md

```diff
@@ -53,9 +68,8 @@ As all the attributes are now symbols, we can:
 * group them in an `AliasSeq`-like fashion to apply them en masse, as is done in LDC for [`@fastmath`](https://github.com/ldc-developers/druntime/blob/ldc/src/ldc/attributes.d#L58)
 * mutate the `AliasSeq`s with compile-time computations.
 
-A module declaration may be tagged with zero or more attribute groups to apply to all symbols declared within the module and acting as the default. Templates are exceptioned and shall remain inferred with explicit tagging. Any attribute groups not provided will be inserted into the module's attribute list from the new `core.attribute.defaultAttributeSet`, the master set of atributes. 
-
-The values of `core.attribute.defaultAttributeSet` will be determined by version conditions under the control of compiler switches to provide maximum utility and flexibility, like so:
+The default value of the core attributes shall be taken from `core.attribute.defaultAttributeSet`.
+The values of `core.attribute.defaultAttributeSet` _may_ be determined by version conditions under the control of compiler switches to provide maximum utility and flexibility, like so:
```

Comment:
That requires recompilation of all libraries (incl. phobos and druntime) with the same version switches, i.e. you cannot use the phobos libs that come with a release.

Reply:
Dependencies are likely compiled separately, so you cannot see their druntime enum values when compiling your application. I'm unclear about the use-case for this, could you elaborate or give a concrete example?
If, as you say, something is rarely needed, to only include it when it's really needed, rather than bloating a proposal.